client/my-sites/domains/domain-search/site-redirect.jsx

```diff
@@ -78,7 +77,9 @@ class SiteRedirect extends Component {
 					{ translate( 'Redirect a Site' ) }
 				</HeaderCake>
 
-				<SiteRedirectStep products={ productsList } selectedSite={ selectedSite } />
+				{ selectedSite && (
```

Comment:
Calypso would crash when going through testing instructions of this PR without this change.

Reply:
Makes sense. It's also a required prop in the `SiteRedirectStep` component.